WordPress/Classes/ViewRelated/Settings/AboutViewController.swift

```diff
@@ -99,23 +95,20 @@ public class AboutViewController : UITableViewController
         
         return cell!
     }
-    
-    public override func tableView(tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+
+    public override func tableView(tableView: UITableView, titleForFooterInSection section: Int) -> String? {
         let isLastSection = section == (rows.count - 1)
-        if isLastSection == false || footerView == nil {
-            return CGFloat.min
+        if !isLastSection {
+            return nil
         }
-        
-        let height = WPTableViewSectionHeaderFooterView.heightForFooter(footerView!.title, width: view.frame.width)
-        return height + footerBottomPadding
+
+        return footerText()
     }
 
-    public override func tableView(tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
-        if section != (rows.count - 1) {
-            return nil
-        }
-        
-        return footerView
+    public override func tableView(tableView: UITableView, willDisplayFooterView view: UIView, forSection section: Int) {
+        guard let view = view as? UITableViewHeaderFooterView else { return }
```

Comment:
Nitpicky, but for code style maybe spread the else clause out over multiple lines. The rationale is we don't do single line if or else clause elsewhere for readability and the same argument can be made for the else clause in a guard statement.